plugins/modules/ipa_group.py

```diff
@@ -116,6 +126,28 @@
     ipa_user: admin
     ipa_pass: topsecret
 
+- name: Add external user to a group
+  community.general.ipa_group:
+   name: developers
+   external: true
+   append: true
+   external_user:
+   - S-1-5-21-123-1234-12345-63421
+   ipa_host: ipa.example.com
+   ipa_user: admin
+   ipa_pass: topsecret
+
+- name: Add a user from MYDOMAIN
+  community.general.ipa_group:
+   name: developers
+   external: true
+   append: true
+   external_user:
+   - MYDOMAIN\john
```

Comment:
```suggestion
   - MYDOMAIN\\john
```

Reply:
(We might also want to use raw Python multiline strings instead of regular Python multiline strings for documentation and example, then this double escaping isn't needed.)